mlflow/store/_unity_catalog/registry/rest_store.py

```diff
@@ -440,6 +446,29 @@ def _get_job_run_id(self, run):
             return None
         return run.data.tags.get(MLFLOW_DATABRICKS_JOB_RUN_ID, None)
 
+    def _get_lineage_input_sources(self, run):
+        if run is None:
+            return None
+        securable_list = []
+        if run.inputs is not None:
+            for dataset in run.inputs.dataset_inputs:
```

Comment:
Can we check upfront whether there are more inputs than the max supported & log a warning? And avoid making a `GetTable` per table?